dicom/@matRad_DicomExporter/matRad_exportDicom.m

```diff
@@ -0,0 +1,49 @@
+function obj = matRad_exportDicom(obj)
+% matRad function to export current workspace to DICOM. Function of
+% matRad_DicomExporter
+% 
+% call
+%   matRad_DicomExporter.matRad_exportDicom()
+%
+% References
+%   -
+%
+% %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%
+% Copyright 2019 the matRad development team. 
+% 
+% This file is part of the matRad project. It is subject to the license 
+% terms in the LICENSE file found in the top-level directory of this 
+% distribution and at https://github.com/e0404/matRad/LICENSES.txt. No part 
+% of the matRad project, including this file, may be copied, modified, 
+% propagated, or distributed except according to the terms contained in the 
+% LICENSE file.
+%
+% %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+%Name of Patient & Study
+%CT Series
+obj = matRad_exportDicomCt(obj);
+
+%RTStruct Series
+
+if ~isempty(obj.cst)
+   obj = matRad_exportDicomRTStruct(obj);    
+end
+
+%if ~isempty(obj.pln)
+%    obj = matRad_exportDicomRTPlan(obj);
+%end
```

Comment:
Because there is not yet a function rt plan export. I recall having a short discussion about that a while back, and we said to leave it out in the beginning, because it entails a lot of machine specific information?